build/tutorials/platform/getting-started-for-users.md

```diff
@@ -0,0 +1,121 @@
+
+You can find a general overview of the Avalanche Platform [here](https://support.avax.network/en/articles/4135427-avalanche-platform-overview), it will be useful in understanding commonalities and differences between Avalanche and other platforms.
+
+## Funding
+
+As all fees on Avalanche are paid in native token, you will need to have some in order to participate in the Avalanche network. You can get it through [exchanges](https://ecosystem.avax.network/marketplace?tag=exchange). Another way to acquire AVAX is via credit card purchase on [Pangolin](https://app.pangolin.exchange/#/buy). Further ways are explained below.
+
+## Wallet
+
+Cryptocurrency funds are usually accessed through wallets. [Avalanche Wallet](https://wallet.avax.network/) is the official web wallet for the Avalanche platform. You can use it to [create your wallet](https://support.avax.network/en/articles/5315160-creating-a-new-wallet-with-the-avalanche-wallet) for use on the network.
+
+You can also use a [HW Ledger](https://docs.avax.network/build/tutorials/platform/setup-your-ledger-nano-s-with-avalanche) to log into your wallet. Using a HW wallet is the most secure way of accessing the network because your private keys and the passphrase never leave the device.
+
+Once you have your wallet, you may want to send your AVAX [from exchange to your wallet](https://support.avax.network/en/articles/5315157-how-to-send-avax-from-an-exchange-to-the-avalanche-wallet).
+
+## Metamask
+
+Most of the activity on the Avalanche network happens on various Dapps (decentralized apps). To interact with them, you need a browser extension that will connect your wallet with the dapp. [Metamask](http://metamask.io/) is one such popular wallet extension.
+
+By default, Metamask connects to Ethereum. To connect to Avalanche, you need to [add Avalanche as a custom network](https://support.avax.network/en/articles/4626956-how-do-i-set-up-metamask-on-avalanche).
+
+In Metamask you can create a new account and send funds to it from your main Avalanche wallet, or import the existing Avalanche Wallet account. You can import the account either by using the secret passphrase or exporting the C-Chain private key from the wallet (Select `Manage Keys`, then `View C Chain Private Key`). If you use the Ledger hardware wallet, you can use it in Metamask too, it will connect to the same wallet.
+
+## Explorers
+
+Explorers are websites that index and present network activity, where you can look up individual transactions, and find out more about what's flowing through the network.
```

Comment:
I think the explorers section(s) should be lower priority on this doc.

Reply:
moved lower.